apps/meteor/client/components/UserInfo/index.ts

```diff
@@ -1,13 +1,7 @@
-import InfoPanel from '../InfoPanel';
+import { InfoPanelLabel, InfoPanelText } from '../InfoPanel';
 import UserInfo from './UserInfo';
 import UserInfoAction from './UserInfoAction';
 import UserInfoAvatar from './UserInfoAvatar';
 import UserInfoUsername from './UserInfoUsername';
 
-export default Object.assign(UserInfo, {
-	Action: UserInfoAction,
-	Avatar: UserInfoAvatar,
-	Info: InfoPanel.Text,
-	Label: InfoPanel.Label,
-	Username: UserInfoUsername,
-});
+export { UserInfo, UserInfoAction, UserInfoAvatar, InfoPanelText as UserInfoText, InfoPanelLabel as UserInfoLabel, UserInfoUsername };
```

Comment:
Why not export directly? Like:

`export { default as UserInfo } from './UserInfo';`

And why InfoPanel components are importing to export with another name? How about to have one single generic component on top of both places used by the User and Panel? InfoText and InfoLabel?